membership_file/admin.py

```diff
@@ -60,25 +69,30 @@ def get_export_filename(self, request, queryset, file_format):
             filename_prefix = "HAS_DEREGISTERED_MEMBERS-"
 
         date_str = datetime.now().strftime('%Y-%m-%d')
-        filename = "%sMembershipFile-%s.%s" % (filename_prefix, date_str, file_format.get_extension())
+        filename = "%sMembershipFile-%s.%s" % (
+            filename_prefix, date_str, file_format.get_extension())
 
         return filename
 
     ##############################
     form = AdminMemberForm
     save_on_top = True
 
-    list_display = ('id', 'user', 'first_name', 'tussenvoegsel', 'last_name', 'educational_institution', 'is_deregistered', 'marked_for_deletion')
+    list_display = ('id', 'user', 'first_name', 'tussenvoegsel', 'last_name',
+                    'educational_institution', 'is_deregistered', 'marked_for_deletion')
     list_filter = [
         'memberyear',
         'is_deregistered', 'marked_for_deletion',
         'is_honorary_member',
         'educational_institution',
-        ('tue_card_number', admin.EmptyFieldListFilter), ('external_card_number', admin.EmptyFieldListFilter),
-        ('key_id', admin.EmptyFieldListFilter), ('phone_number', admin.EmptyFieldListFilter),
+        ('tue_card_number', admin.EmptyFieldListFilter), ('external_card_number',
+                                                          admin.EmptyFieldListFilter),
+        ('key_id', admin.EmptyFieldListFilter), ('phone_number',
+                                                 admin.EmptyFieldListFilter),
```

Comment:
It's neater to place the enter outside of the brackets, than inside in this case.

```suggestion
        ('tue_card_number', admin.EmptyFieldListFilter),
        ('external_card_number', admin.EmptyFieldListFilter),
        ('key_id', admin.EmptyFieldListFilter),
        ('phone_number', admin.EmptyFieldListFilter),
```